daemon/metrics.go

```diff
@@ -26,12 +26,13 @@ func init() {
 		containerActions.WithValues(a).Update(0)
 	}
 	networkActions = ns.NewLabeledTimer("network_actions", "The number of seconds it takes to process each network action", "action")
-	engineVersion = ns.NewLabeledGauge("engine", "The version and commit information for the engine process", metrics.Unit("info"),
+	engineInfo = ns.NewLabeledGauge("engine", "The information related to the engine and the OS it is running on", metrics.Unit("info"),
 		"version",
 		"commit",
 		"architecture",
-		"graph_driver", "kernel",
-		"os",
+		"graphdriver",
+		"kernel", "os",
+		"os_type", "id", // ID is a randomly generated unique identifier (e.g. UUID4)
```

Comment:
Something more descriptive than "id" would be useful.  There's a non-trivial chance such a label will already exist as a target label for some users, which will turn it into `external_id` by default.